int32 loc_ant